api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerEntitySoundEffect.java

```diff
@@ -50,6 +59,9 @@ public void read() {
         this.entityId = readVarInt();
         this.volume = readFloat();
         this.pitch = readFloat();
+        if (serverVersion.isNewerThanOrEquals(ServerVersion.V_1_19)) {
+            this.seed = readLong();
```

Comment:
https://wiki.vg/Protocol#Entity_Sound_Effect

is protocol wiki wrong?  I don't see this

Reply:
No you are right, sleepy eyes